Dockerfile

```diff
@@ -106,7 +106,7 @@ RUN yum -y install ${SCL_HTTPD_VERSION} ${SCL_PYTHON_VERSION}-mod_wsgi && \
 
 # Copy the cfgov directory form the build image
 COPY --from=cfgov-build --chown=apache:apache ${CFGOV_PATH}/cfgov ${CFGOV_PATH}/cfgov
-COPY --from=cfgov-build --chown=apache:apache ${CFGOV_PATH}/docker-entrypoint.sh ${CFGOV_PATH}/docker-entrypoint.sh
+COPY --from=cfgov-build --chown=apache:apache ${CFGOV_PATH}/docker-entrypoint.sh ${CFGOV_PATH}/refresh-data.sh ${CFGOV_PATH}/
```

Comment:
Is the database dump file being downloaded using the env var as part of the refresh-data.sh script? If so, I think you don't need the path argument to it? Unless it's a working directory issue in the container or something I'm not familiar with.

Reply:
this is dropping it into the docker container, if we don't provide the path here at the end then it puts it in the home directory of the prod build and we wouldn't see it.